client/src/reducers/getMealReviewsReducer.js

```diff
@@ -0,0 +1,16 @@
+import {GET_MEAL_REVIEWS} from "../actions/actionTypes"
+
+const initialState = {};
+
+export default function (state = initialState,action) {
+  switch(action.type) {
+    case `${GET_MEAL_REVIEWS}_FULFILLED`:
```

Comment:
Pending should be the first case for readability.

Reply:
Fixed